tools/metrics/import/import-prometheus-s3.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+usage() {
+    # TODO: Optionally being able to provide a different path to save the files to other than default
+    echo "Usage: $0 [--profile AWS_PROFILE] -f FILE_PATH"
+    exit 1
+}
+
+# Initialize variables
+AWS_PROFILE=""
+FILE_PATH=""
+
+# Parse arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --profile) AWS_PROFILE="$2"; shift;;
+        -f) FILE_PATH="$2"; shift ;;
+        *) usage ;;
+    esac
+    shift
+done
+
+echo "Using AWS_PROFILE=$AWS_PROFILE"
+
+if [[ -z "$FILE_PATH" ]]; then 
+    echo "Need to supply the s3 path to the prometheus data"
+    usage 
+fi
+
+prom_data_dir=$(dirname "$(realpath "$0")")
+
+mkdir -p "$prom_data_dir/temp"
+
+# Delete and recreate the data directory
+if [[ -d "$prom_data_dir/data" ]]; then
+    echo "Data directory exists. Attempting to delete..."
+    sudo rm -rf "$prom_data_dir/data"
+    if [[ $? -ne 0 ]]; then
+        echo "Error: Failed to delete the data directory. Check permissions."
+        exit 1
+    fi
+fi
```

Comment:
Does it mean we could only have a single prometheus post-mortem dataset ? If so it should be changed because it could be interesting to compare multiple datasets.

Reply:
You'd have to deploy different Prometheus deployments for different data and run the same script for the different deployments. But because filtering between them would be complicated it would require additional work to add labels relevant to the different Prom deployments. We'd probably have the same problems related to backfilling as mentioned [in my second point here](https://publish.obsidian.md/sleep/Projects/ArmoniK/Tasks/Explore+exporting+Prometheus+through+Fluentbit) 

For the future goal of comparing different ArmoniK deployments (multiple Prometheus deployments), we could have that discussion/look into it separately.